content/plugins/scottboms/kirbytag-svg/plugin.txt

```diff
@@ -26,6 +26,8 @@ Example:
 
 ----
 
+<<<<<<< HEAD
```

Comment:
Looks you still have leftovers of a merge conflict in here

Reply:
Oh, will have a look. Sorry about that though I fixed all that.